bucket/jstock.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://jstock.org",
+    "description": "JStock is a free stock market software for 28 countries",
+    "license": "GPL-2.0-only",
+    "version": "1.0.7.45",
+    "extract_dir": "jstock",
+    "architecture": {
+        "64bit": {
```

Comment:
What is behind your assumption this is 64bit only?

https://github.com/yccheok/jstock/blob/ba607c28beb780936bbf23abdc61dc2b6de9f072/installer/windows/jstock.xml#L24

Reply:
I have not tested it on 32bit so I don't assume it works.